charts/tempo-distributed/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: tempo-distributed
 description: Grafana Tempo in MicroService mode
 type: application
-version: 1.17.0
+version: 1.18.0
 appVersion: 2.5.0
```

Comment:
the app version should be bumped, when it is clear which release `log_discarded_spans` will be part of. Check the [CHANGELOG.md](https://github.com/grafana/tempo/blob/main/CHANGELOG.md) for 

> [FEATURE] Discarded span logging log_discarded_spans https://github.com/grafana/tempo/issues/3957 (@dastrobu)